build.sh

```diff
@@ -16,6 +16,7 @@
 git submodule update --init --recursive --remote
 mkdir -p build
 cd build
+find . -name "CMakeCache.txt" -delete
```

Comment:
Can we remove this? I think Jun Hao has a better way to delete cached files on AWS.

Reply:
This is still not addressed